apps/web/src/pages/Pools/IndexPoolPage/IndexPoolPage.tsx

```diff
@@ -42,7 +49,7 @@ export const PoolPage = () => {
     <AdminContentWrapper crumbs={navigationCrumbs}>
       <SEO title={pageTitle} />
       <PageHeader icon={Squares2X2Icon}>{pageTitle}</PageHeader>
-      <PoolTableApi />
+      {isAdmin ? <PoolAdminTableApi /> : <PoolOperatorTableApi />}
```

Comment:
I suggest pulling in the isLoaded field from useAuthorization and maybe use Pending so you don't render either of these until auth is settled.  It's impossible to get here without auth since auth is also used for routing but instead of depending on that to never change let's defend this page.

Reply:
like so https://github.com/GCTC-NTGC/gc-digital-talent/pull/6080/commits/798bb235a890746d527b2ddaff07f97074fa7d98 ?

Reply:
Nice, that looks great!